README.md

```diff
@@ -329,6 +370,11 @@ You can then run the following two commands in order to be able to connect to Pr
   kubectl --namespace prometheus port-forward $POD_NAME 9090
   ```
 
+This may fail with a warning about status being "Pending" until Prometheus has started, retry once status is "Running" for all pods:
+  ```sh
+  watch -n1 kubectl -n prometheus get pods
```

Comment:
It probably does exactly the same under the covers - but perhaps it's worth using the `kubctl` version of the commands everywhere: 
`kubectl get pods --watch`

Reply:
done